Journal/Journal/EntriesTableViewController.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  EntriesTableViewController.swift
+//  Project
+//
+//  Created by Jonathan Ferrer on 6/3/19.
+//  Copyright © 2019 Jonathan Ferrer. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+
+
+class EntriesTableViewController: UITableViewController, NSFetchedResultsControllerDelegate{
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.reloadData()
+
+    }
+
+    // MARK: - Table view data source
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        tableView.reloadData()
```

Comment:
You don't need to reload table here this way. NSFetchedResultsControllerDelegate is taking care of that for you.